.example.env

```diff
@@ -17,4 +17,12 @@ REACT_APP_PINATA_GATEWAY_HOSTNAME=
 REACT_APP_BLOCKNATIVE_API_KEY=
 
 # sentry url. Not needed for development
-REACT_APP_SENTRY_DSN=
\ No newline at end of file
+REACT_APP_SENTRY_DSN=
+
+# Secret phrase used for e2e testing with metamask.
+# !!-- ** USE A TEST WALLET, NOT YOUR REAL WALLET!!!! ** -- !! 
+E2E_SECRET_WORDS=
+# Password used for e2e testing with metamask. Set this to whatever you prefer.
+E2E_PASSWORD=
```

Comment:
**suggesiton** I got a bit stuck here. WDYT about adding easily reproducible steps for generating/creating these values to the README?

Reply:
Absolutely - as discussed out of band I think we can add instructions to how to generate a BIP39 mnemonic seed phrase to be used here effortlessly